instance/models/openedx_instance.py

```diff
@@ -84,6 +86,8 @@ class OpenEdXInstance(Instance, OpenEdXAppConfiguration, OpenEdXDatabaseMixin,
     external_lms_preview_domain = models.CharField(max_length=100, blank=True)
     external_studio_domain = models.CharField(max_length=100, blank=True)
 
+    load_balancing_server = models.ForeignKey(LoadBalancingServer, null=True, blank=True, on_delete=models.PROTECT)
```

Comment:
I noticed that it's actually quite easy to make `LoadBalancingServer` completely generic by using Django model reflection to find the subclasses of `Instance` that refer to `LoadBalancingServer`, so I implemented this now.  To make a different subclass of instance load-balanced, all that's needed is to add a `ForeignKey` field like this one here and implement `get_load_balancer_configuration()`.

Does this settle this point, or do you think I should add a mixin class containing only this field?